### PR TITLE
[16.0][ADD] pos_config_logo: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Configure a different logo in selected points of sale.

cc @Tecnativa TT50219

please review @pedrobaeza @CarlosRoca13

### CODE REVIEW

File: pos_config_logo/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "POS Lot Selection",
```
Comment (carlos-lopez-tecnativa) [inline pos_config_logo/__manifest__.py]: Please check module name

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM tested functionally, only a minor change

**Review by CarlosRoca13 — APPROVED** [review]
Just check the comment of @carlos-lopez-tecnativa

File: pos_config_logo/__manifest__.py (hunk 1 above)
Comment (chienandalu) [reply pos_config_logo/__manifest__.py]: Fixed :sweat_smile: thanks!

**Pull request merged**